providers/flipt/pom.xml

```
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>dev.openfeature.contrib</groupId>
+        <artifactId>parent</artifactId>
+        <version>0.1.0</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    <groupId>dev.openfeature.contrib.providers</groupId>
+    <artifactId>flipt</artifactId>
+    <version>0.0.1</version> <!--x-release-please-version -->
+
+    <name>flipt</name>
+    <description>Flipt provider for Java</description>
+    <url>https://www.flipt.io/</url>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.flipt</groupId>
+            <artifactId>flipt-java</artifactId>
+            <version>0.2.13</version>
```

```suggestion
            <version>0.2.14</version>
```

should now be java 8 compat